src/site/content/en/mini-apps/mini-app-standardization/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: Mini app standardization
+authors:
+  - thomassteiner
+date:  2021-02-05
+# updated: 2021-01-25
+description: |
+  This chapter introduces the standardization effort that has been started for mini apps.
+tags:
+  - mini-apps
+---
+
```

Comment:
This one could also use an intro: what you've covered so far and what you'll cover now

Reply:
I have solved this in https://github.com/GoogleChrome/web.dev/pull/3938/commits/9b1f6c5d6f25c7117132deef98f3c44472eca689 by adding an aside to each chapter apart the first that reads:

```
This post is part of an article collection where each article builds upon previous articles. If you just landed here, you may want to start reading from the beginning.
```